functions/consumer/zeromq-consumer/src/main/java/org/springframework/cloud/fn/consumer/zeromq/ZeroMqConsumerConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package org.springframework.cloud.fn.consumer.zeromq;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.zeromq.ZContext;
+import org.zeromq.ZMQ;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.mapping.OutboundMessageMapper;
+import org.springframework.integration.zeromq.outbound.ZeroMqMessageHandler;
+import org.springframework.messaging.Message;
+
+@Configuration
```

Comment:
`@author` and `@since`, please.

Reply:
Should/can this be added to checkstyle?

Reply:
I think the Checkstyle over here is a bit relax according the Spring Cloud foundation, but we can ask @sobychacko what is going on.
I know that Copyright can be validated, but I'm not sure if it is easy to add `@author` check though...